<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <widget class="QWidget" name="Chat">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0" colspan="3">
            <widget class="QLineEdit" name="MessagelineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
	 background: white;
     selection-background-color: darkgray;
 }</string>
             </property>
             <property name="placeholderText">
              <string>Write a message...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QListWidget" name="UserlistWidget">
             <property name="styleSheet">
              <string notr="true">QListView {
     background-color: white;
	  border-radius: 15px;
 }</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QTextBrowser" name="MessagesBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>550</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">  QTextBrowser{
     background-color: white;
	 border-radius: 15px;
 }</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="Send">
             <property name="styleSheet">
              <string notr="true"> QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);

     min-width: 80px;
	 min-height: 25px;
 }</string>
             </property>
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Login">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QFrame" name="LoginFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>550</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"> QFrame {
     border: 0px solid rgb(28, 43, 255);
     border-radius: 15px;
     padding: 2px;
	background-color: rgb(203, 206, 255);

 }</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="EmptyLabel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="UsernameLabel">
                <property name="text">
                 <string>User name</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="UserlineEdit">
                <property name="styleSheet">
                 <string notr="true"> QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
	 background: white;
     selection-background-color: darkgray;
 }</string>
                </property>
                <property name="placeholderText">
                 <string>enter a nickname without the colon character</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="EmptyLabel_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="Sign">
                <property name="text">
                 <string>Sign in</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>UserlineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>Sign</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MessagelineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>Send</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>680</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
